task3/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Здесь нужно запросить у пользователя число.
```suggestion
        {
           Console.WriteLine("Введите число");
            double x = double.Parse(Console.ReadLine());
```